security-analysis/security-analysis-api/src/test/resources/ActionFileTestV1.0.json

```diff
@@ -162,5 +162,10 @@
     "acEmulationEnabled" : false,
     "activePowerSetpoint" : 12.0,
     "relativeValue" : true
+  }, {
+    "type" : "SHUNT_COMPENSATOR_POSITION",
```

Comment:
As discussed, it's a bit strange to have new type of actions in v1.0 (or in v1.1), but it seems that this kind of forward-compatibility is the way the versions are dealt with here.